### ISSUE TITLE
COCO Setup

### ISSUE THREAD

**Issue by Simon4Yan** [Simon4Yan/Meta-set#2]
Thank you for your attention!

Please download the datasets  for coco classification setup in [here](https://anu365-my.sharepoint.com/:u:/g/personal/u6854640_anu_edu_au/EVwMYZdEYodCtRtGeiVR1doBUnExqoYJGOrvLzwFw28j_w?e=znydPh).

The zip file contains two parts. **The first part** is coco datasets: 1) a training set, 2) a validation set, 3) the validation set without background, and 4) validation sets with various backgrounds. 

Some users reported that the COCO creation is slow. Here is an alternative to creating a meta-dataset: applying random image transformations to change the visual characteristics of 4) validation sets with various backgrounds. Given a validation set with a changed background, we can apply 5 random transformations to diversify it.

The users are suggested to use the way of ImageNet-C to apply transformations. ImageNet-C uses Pytorch data loader to speed up the process, please refer to [the code](https://github.com/hendrycks/robustness/blob/8190fe329f5f072a06e3a2aea02bb5dda69aed9f/ImageNet-C/create_c/make_imagenet_c.py#L81). In our works, we use Imgaug for the transformations and there are other corruptions such as [ImageNet-C-Bar](https://github.com/hendrycks/robustness/blob/8190fe329f5f072a06e3a2aea02bb5dda69aed9f/ImageNet-C/create_c/make_imagenet_c.py#L81).

Note that, we provide 3) the validation set without background, so the users can change the background easily based on their usage. 

**The second part** contains three real-world test sets, 1) Pascal, 2) Caltech, and 3) ImageNet (note that, ImageNet test set is from the[ImageCLEF dataset](https://www.imageclef.org/2014/adaptation)). We also provide test sets with some image transformations. Enjoy!

**Comment by Simon4Yan** [comment]
Following up on the above, the learned coco classification model is [here](https://anu365-my.sharepoint.com/:u:/g/personal/u6854640_anu_edu_au/EVx7YpxdsD5MlhjFVHVomWsBb-oiF0CEfKintAv7kEA4Yw?e=Vwqmj0).

**Comment by Simon4Yan** [comment]
If you find our project useful, please cite our works:

```
    @inproceedings{deng2020labels,
    author={Deng, Weijian and Zheng, Liang},
    title     = {Are Labels Always Necessary for Classifier Accuracy Evaluation?},
    booktitle = {Proc. CVPR},
    year      = {2021},
    }
```

```
    @inproceedings{deng2022labels,
    author={Deng, Weijian and Zheng, Liang},
    title     = {Are Labels Always Necessary for Classifier Accuracy Evaluation?},
    booktitle = {TPAMI},
    year      = {2022},
    }
```

Have a nice day!
-Weijian

**Comment by ashygsy** [comment]
link has expired

**Comment by Simon4Yan** [comment]
@ashygsy 
> link has expired

Thanks for the reminder. I have fixed it. OneDrive notices me _"Your organization's policy requires this link to expire after 30 days"_. I will find out a way to maintain the link.

Regards,
Weijian

**Comment by tomvii** [comment]
Hi. Would it be possible to refresh the link again? It seems that the link has expired again. Thank you.

**Comment by Simon4Yan** [comment]
> Hi. Would it be possible to refresh the link again? It seems that the link has expired again. Thank you.

Thanks. I have refreshed, I will use google drive later. Best, Weijian

**Comment by 02zp11** [comment]
Would it be possible to refresh the link again? It seems that the link has expired again. Thank you very much.

**Comment by Simon4Yan** [comment]
Just refreshed, sorry for late response (struggling with CVPR...)
